core/templates/filters/translate.pipe.ts

```diff
@@ -0,0 +1,133 @@
+// Copyright 2019 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Translate pipe for i18n translations.
+ */
+
+import { ChangeDetectorRef, OnDestroy, Pipe, PipeTransform }
+  from '@angular/core';
+import { Subscription } from 'rxjs';
+import { LangChangeEvent, TranslateService } from 'services/translate.service';
+import { UtilsService } from 'services/utils.service';
+
+/**
+ * Commonly used terms in this file.
```

Comment:
The location of this comment is fine, but I mostly don't understand it -- it's not structured well. E.g. it starts with "Commonly used terms in this file" and then goes into something that's not a commonly-used term and then into an "example", which makes this hard to follow because I was expecting to see an actual list of terms coming after the first sentence. If you're introducing an example, then state something like "Here are some commonly-used terms in this file, with reference to the following example: ...". You can put tangential notes at the end.

In general, try and write your comments in complete prose sentences and thoughts, like a paragraph in a book or a well-written explanation. At the moment it reads in a rather scattered way. Try and make it easy for the reader.

Reply:
Changed it now. Is it better?